lib/Elastica/Query/MultiMatch.php

```diff
@@ -144,13 +144,13 @@ public function setType($type)
     /**
      * Set fuzziness.
      *
-     * @param float $fuzziness
+     * @param mixed $fuzziness
```

Comment:
int|string instead of mixed? Or instead of mix argument add method like autoFuzziness and left setFuzziness as is?